inductiva/tasks/methods.py

```diff
@@ -49,3 +50,45 @@ def fetch_task_output(task_id: str,
             result_list = api.download_output(api_instance, task_id, output_dir)
 
             return unpack_output(result_list, output_dir, return_type)
+
+
+def get_task(task_id) -> Dict:
```

Comment:
Let's iterate a bit on the naming to be more specific and to clarify the reason to have just a `get_task_status`. Suggestion: `get_task_info`?